packages/translations/locales/en/admin.json

```
@@ -1532,5 +1533,135 @@
+    },
+    "expiring": {
+      "title": "Expiring Soon",
+      "noSubscriptions": "No subscriptions expiring soon"
+    },
+    "modal": {
+      "pause": {
+        "title": "Pause Subscription",
+        "description": "Pause this subscription. The subscriber will lose access until resumed."
+      },
+      "cancel": {
+        "title": "Cancel Subscription",
```

Biome (2.1.2)

[error] The key cancel was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)